flink-runtime/src/main/java/org/apache/flink/streaming/runtime/tasks/mailbox/TaskMailbox.java

```diff
@@ -128,7 +134,7 @@ public interface TaskMailbox {
      * not affect {@link #tryTake(int)} and {@link #take(int)}; that is, they return the same mails
      * even if no batch had been created.
      *
-     * <p>The default batch is empty. Thus, this method must be invoked once before {@link
+     * <p>By default, the batch is empty. Thus, this method must be invoked once before {@link
```

Comment:
`The default batch is empty.` -> `By default, the batch is empty.`. The latter is more accurate.